docs/src/main/asciidoc/cdi.adoc

```diff
@@ -167,13 +167,15 @@ This bean would be assignable to `@Inject @Superior Translator` and `@Inject @Su
 The reason is that `@Inject Translator` is automatically transformed to `@Inject @Default Translator` during typesafe resolution.
 And since our `SuperiorTranslator` does not declare `@Default` only the original `Translator` bean is assignable.
 
-== Looks good. What is the bean scope?
+[[bean-scope]]
+== [[looks-good-what-is-the-bean-scope]] Looks good. What is the bean scope?
```

Comment:
What's the `[[looks-good-what-is-the-bean-scope]]` for? I mean the `[[bean-scope]]` id is enough, or?

Reply:
That's so I don't break existing links to that anchor.

Reply:
I see. Well, I'd probably just use the `[[looks-good-what-is-the-bean-scope]]` as an explicit id but this is also ok... ;-).